cyfronet-fid/sat4envi
# [#252] Sync params with url

## What is in this PR?

Synchronisation between URL's query params, store and interactive map

## How to test?

Move / zoom map around, change visible layers & products. Copy URL, open new private window and paste URL there - the map, layers, product in the new window should be set to be the same as in the old one.

[Closes #252]
